examples/readiness-probe-pod

```diff
@@ -0,0 +1,19 @@
+# Copy of pod.yaml without file extension for test
```

Comment:
Why not modify docs/user-guide/pod.yaml to add the probe?

On Monday, February 8, 2016, Prashanth B <notifications@github.com> wrote:

> In examples/readiness-probe-pod
> <https://github.com/kubernetes/kubernetes/pull/20865#discussion_r52259527>
> :
>
> > @@ -0,0 +1,19 @@
> > +# Copy of pod.yaml without file extension for test
>
> Yeah this is why I'd suggested just adding the probe inline. If you move
> to test/ you'll need to make sure the directory you put it in is included
> in the test tarball. @ixdy <https://github.com/ixdy> where would be a
> safe place to put yaml? IIRC the last time I shoved something in
> test/e2e/testing-manifests it wasn't available on the server, and I ended
> up just inling it anyway
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/20865/files#r52259527>.
>